testsuite/features/support/env.rb

```diff
@@ -18,9 +18,9 @@
 
 server = ENV['SERVER']
 $debug_mode = true if ENV['DEBUG']
-$long_tests_enabled = true if ENV['LONG_TESTS']
+$long_tests_enabled = true if ENV['LONG_TESTS'] == 'true'
 puts "Executing long running tests" if $long_tests_enabled
-$service_pack_migration_enabled = true if ENV['SERVICE_PACK_MIGRATION']
+$service_pack_migration_enabled = true if ENV['SERVICE_PACK_MIGRATION'] == 'true'
 # To disable SP migration, comment the previous line, uncoment the next one
 # and adjust testsuite/run_sets/init_clients.yml
```

Comment:
is it still needed to adjust that yaml file?

Reply:
It seems not.  Good catch.